tests/main/interfaces-system-files/task.yaml

```diff
@@ -0,0 +1,91 @@
+summary: Ensure that the system-files interface works.
+
+details: |
+    The system-files interface allows access specific system files or directories.
+
+prepare: |
+    # shellcheck source=tests/lib/snaps.sh
+    . "$TESTSLIB/snaps.sh"
+    install_local test-snapd-sh
+
+    # shellcheck source=tests/lib/files.sh
+    . "$TESTSLIB/files.sh"
+
+    # Fist layer of dirs and files
+    ensure_file_exists_backup_real /tmp/.testfile1
+    ensure_file_exists_backup_real /tmp/testfile1
+    ensure_dir_exists_backup_real /tmp/.testdir1
+    ensure_dir_exists_backup_real /tmp/testdir1
+
+    # Second layer of dirs and files
+    ensure_file_exists_backup_real /tmp/.testdir1/.testfilé2
+    ensure_file_exists_backup_real "/tmp/.testdir1/test file2"
+    ensure_dir_exists_backup_real /tmp/root
```

Comment:
Similarly, these are not good test cases since /tmp should be allowed (see above).